src/ViewHelper.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require_once 'src/Items.php';
+
+class ItemViewHelper
```

Comment:
As per my comment on the Item class, you should rename this to be more specific `TrainerViewHelper` for example